protobuf/v1/registration.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package v1.registration;
+
+// register the dataplane node with control-plane
+message RegisterRequest {
+  // id of the mayastor instance
+  string id = 1;
+  // grpc endpoint of the mayastor instance
+  string grpc_endpoint = 2;
```

Comment:
can we add a "instance_uuid" to the registration? This would help the control plane determine if mayastor instance has restarted or not.